networkx/algorithms/components/strongly_connected.py

```diff
@@ -371,6 +371,26 @@ def condensation(G, scc=None):
     NetworkXNotImplemented
         If G is undirected.
 
+    Examples
+    -----
+    >>> G = nx.barbell_graph(4, 0)
+    >>> G.remove_edge(3, 4)
+    >>> G = nx.DiGraph(G)
```

Comment:
Thanks for this!
Could you also add a quick comment about the graph construction. I wrote down the example
quickly without a comment. Could you put something like `# create 2 distinct SCC using the barbell graph`